test/inputoutput/test_io.py

```diff
@@ -1,3 +1,13 @@
-from main.inputoutput.io import inc
+from main.inputoutput.io import inc,unzip_file
+import os
+
 def test_answer():
     assert inc(3) == 5
+
+def test_unzip_file():
+    fromPath=r"test\data\Desktop.zip"
+    toPath=r"test\data\target"
+    curDir=os.getcwd()
```

Comment:
Tests should be able to work from any location, getting current directory and then adding path may not work always. Need to think of better solution. Maybe use absolute path instead of relative path? Not sure